Demo/EarlGreyExample/EarlGreyContribSwiftTests/GetText.swift

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Example Usage:
+ *
+ *  let element = Element()
+ *
+ *  domainField.performAction(grey_typeText("hello.there"))
+ *             .performAction(grey_getText(element))
+ *
+ *  GREYAssertTrue(element.text != "", reason: "get text failed")
+ */
+func grey_getText(elementCopy: Element) -> GREYActionBlock {
```

Comment:
Is there any reason why these are Swift only classes? We can add these to GREYActions or so and have them exported in the exact same way that all other matchers are. 

Also, I'm not completely sure we need an entirely new project for this. In the FunctionalTests.xcodeproj, could we not add the view controller and tests for these methods with a separate of "EarlGreyExperimentalTests.m" file or so?